pkgs/misc/emulators/yuzu/default.nix

```diff
@@ -4,7 +4,7 @@ let
 in {
   mainline = libsForQt5.callPackage ./base.nix rec {
     pname = "yuzu-mainline";
-    version = "844";
+    version = "878";
     branchName = branch;
     src = fetchFromGitHub {
```

Comment:
It looks like you didn't include the changes to `fetchFromGitHub.sha256`

Reply:
I updated the hash in the latest revision.